src/qrl/core/txs/TransferTransaction.py

```diff
@@ -9,7 +9,7 @@
 
 class TransferTransaction(Transaction):
     """
-    SimpleTransaction for the transaction of QRL from one wallet to another.
+    This transaction sends QRL from one address to another.
```

Comment:
1. "QRL" <-- Replace QRL by Quanta.
2. Also worth to mention, that, it can have maximum 100 output address per transaction. So it allows to send coins to multiple addresses in one transaction.